lib/sisimai/reason/userunknown.rb

```diff
@@ -158,7 +158,7 @@ def true(argvs)
           return true if argvs.reason == 'userunknown'
 
           require 'sisimai/smtp/status'
-          diagnostic = argvs.diagnosticcode;
+          diagnostic = argvs.diagnosticcode.downcase;
```

Comment:
__[RuboCop]__

Style/Semicolon: Do not use semicolons to terminate expressions.


<!-- Comment by [SideCI](https://sideci.com) -->